inception/inception-experimental-editor/src/main/java/de/tudarmstadt/ukp/inception/experimental/editor/resources/AnnotationExperienceAPIBasicEditor.js

```diff
@@ -0,0 +1,1644 @@
+    textElement.style.display = "block";
+    let xBegin = 0;
+    this.sentenceBegin = Number(document.getElementsByTagName("input")[2].value) - 1;
+    if (this.sentenceBegin + this.sentenceCount > sentences.length) {
+      this.sentenceCount = sentences.length - this.sentenceBegin;
+    }
+    if (this.showBackground) {
+      svg.appendChild(this.drawBackground());
+    }
+    if (this.showSentenceNumbers) {
+      svg.appendChild(this.drawSentenceNumbers());
+      textElement.style.width = (svg.width - this.SENTENCE_OFFSET_WIDTH).toString();
+      xBegin = this.SENTENCE_OFFSET_WIDTH;
+    }
+    let k = 0;
+    let xPrev = xBegin;
+    for (let i = 0; i < this.sentenceCount; i++) {
+      let sentenceOffset = this.calculateInitialOffset(this.sentenceBegin + i);
+      let sentence = document.createElementNS("http://www.w3.org/2000/svg", "g");
+      sentence.setAttribute("class", "text-row");
+      sentence.setAttribute("sentence-id", (this.sentenceBegin + i).toString());
+      for (let j = 0; j < sentences[this.sentenceBegin + i].length; j++, k++) {
+        let char = document.createElementNS("http://www.w3.org/2000/svg", "text");
+        char.textContent = sentences[this.sentenceBegin + i][j];
+        char.setAttribute("x", xPrev.toString());
+        char.setAttribute("y", ((i + 1) * 20 - 5).toString());
+        char.setAttribute("char_pos", (sentenceOffset + j).toString());
+        xPrev += this.CHARACTER_WIDTH;
+        sentence.appendChild(char);
+      }
+      textElement.appendChild(sentence);
+      xPrev = xBegin;
+    }
+    svg.appendChild(textElement);
+    textArea.appendChild(svg);
+    if (this.showHighlighting) {
+      svg.appendChild(this.drawAnnotation(this.annotationExperienceAPIBasicEditor.annotationExperienceAPI.spans, aElementId));
+    }
+  }
+  drawBackground() {
+    let background = document.createElementNS("http://www.w3.org/2000/svg", "g");
+    background.setAttribute("class", "background");
+    background.innerHTML = "";
+    for (let i = 0; i < this.sentenceCount; i++) {
+      let rect = document.createElementNS("http://www.w3.org/2000/svg", "rect");
+      rect.setAttribute("x", "0");
+      rect.setAttribute("y", (i * 20).toString());
+      rect.setAttribute("width", "100%");
+      rect.setAttribute("height", "20");
+      if (i % 2 == 0) {
+        rect.setAttribute("fill", this.lineColorFirst);
+      } else {
+        rect.setAttribute("fill", this.lineColorSecond);
+      }
+      background.appendChild(rect);
+    }
+    return background;
+  }
+  calculateInitialOffset(aSentenceNumber) {
+    let offset = 0;
+    let sentences = this.annotationExperienceAPIBasicEditor.annotationExperienceAPI.viewport.documentText.split(".");
+    for (let i = 0; i < aSentenceNumber; i++) {
+      let words = sentences[i].split(" ");
+      for (let j = 0; j < words.length; j++) {
+        offset += words[j].length + 1;
+      }
+      offset++;
+    }
+    return offset;
+  }
+  drawSentenceNumbers() {
```

Comment:
In the brat module, we use the JQuery SVG library for working more nicely with SVG data (cf. JQuerySvgDomResourceReference.java, JQuerySvgResourceReference.java).